MobiusExtras/Test/ConnectableClassTests.swift

```diff
@@ -1,148 +0,0 @@
-// Copyright (c) 2019 Spotify AB.
-//
-// Licensed to the Apache Software Foundation (ASF) under one
-// or more contributor license agreements.  See the NOTICE file
-// distributed with this work for additional information
-// regarding copyright ownership.  The ASF licenses this file
-// to you under the Apache License, Version 2.0 (the
-// "License"); you may not use this file except in compliance
-// with the License.  You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing,
-// software distributed under the License is distributed on an
-// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-// KIND, either express or implied.  See the License for the
-// specific language governing permissions and limitations
-// under the License.
-
-import MobiusCore
-import Nimble
-import Quick
-
-@testable import MobiusExtras
-
-class ConnectableTests: QuickSpec {
```

Comment:
I’d prefer not to delete tests of deprecated stuff – if it’s available, it should be tested. In at least one of my working branches I disabled warnings-as-errors in order to keep deprecated tests around for now

Reply:
I see. I only removed these to comply with the warnings-as-errors check. I will re-add these tests and disable the check instead.